modules/src/ZeroTier/scripts/dependencies.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# 2023 - m5kro
+
+[[ -f /tmp/zerotier.progress ]] && {
+  exit 0
+}
+
+touch /tmp/zerotier.progress
+
+if [ "$1" = "install" ]; then
+  if [ "$2" = "internal" ]; then
+     opkg update
+     opkg install kmod-tun
+     opkg install libmnl
+     opkg install ip-tiny
+     opkg install libminiupnpc
+     opkg install libnatpmp
+     opkg install zerotier
+
+# kmod-tun installed at root due to /dev/net/tun missing issues
+  elif [ "$2" = "sd" ]; then
+     opkg update
+     ln -s /sd/modules/ZeroTier /pineapple/modules/
+     opkg install kmod-tun
+     opkg -d sd install libmnl
+     opkg -d sd install ip-tiny
+     opkg -d sd install libminiupnpc
+     opkg -d sd install libnatpmp
+     opkg -d sd install zerotier
+     cp /sd/etc/config/zerotier /etc/config/
```

Comment:
I understand that all this block would be unnecessary because these directories are already linked to contemplate these cases.
The link is made here:
```
./etc/php7-fpm.d/www.conf:365
./etc/uci-defaults/97-pineapple.sh:7
```

Reply:
Not sure what you are referring to here. 

If it is about all the linked files, opkg does not do it when installing to sd. This causes issues for ZeroTier because it does not try to look in sd for dependencies. 

As for the zerotier config file, opkg also doesn't link it and uci does not look in sd. So when setting uci values it errors.